asf_heat_pump_suitability/pipeline/suitability/prototype_calculate_suitability.py

```diff
@@ -0,0 +1,396 @@
+"""
+Functions to calculate suitability of different HP technologies.
+"""
+
```

Comment:
happy to just delete this file! It'll be in the github records anyway if for some reason its needed